build/ProjectReunion-BuildFoundation.yml

```diff
@@ -391,14 +391,6 @@ jobs:
 #    dependsOn: CreateNugetPackage
 #    pkgArtifactPath: '$(artifactDownloadPath)\fullnuget\FrameworkPackage'
```

Comment:
Delete.

Reply:
Good point.
It looks like the solution doesn't exist either (`test\ProjectReunion-ReleaseTest`). I'll remove this and the related files.
Thanks!